src/app/calendar/parcelCalendarFunctions.ts

```diff
@@ -28,12 +50,12 @@ export const parcelsToCollectionEvents = (
     colorMap: LocationColorMap
 ): CalendarEvent[] => {
     return parcels.map((parcel) => {
-        const fullName = parcel.clients?.full_name ?? "";
+        const fullName = parcel.client!.full_name ?? "";
```

Comment:
I don't think this change does what you want

Reply:
I talked to Tom and he said a parcel must be associated with a client and if we delete a client the parcel will be deleted cascade